assets/texts/de/privacy.txt

```diff
@@ -4,6 +4,6 @@ Eine Erklärung aller Berechtigungen, die dieses Add-on erfragt, kann auf https:
 
 == DIENSTE VON DRITTEN ==
 
-Dieses ADD-ON nutzt den „Sync storage” des Browser's um die Einstellungen zu speicheren. Wenn der NUTZER „Sync” im Browser aktiviert, werden die Einstellungen des Add-ons hoch geladen und zwischen den Geräten, die mit dem (Mozilla) Account des Browsers verbunden sind, synchronisiert. Wenn dies nicht gemacht wird, werden die Daten lokal auf dem Gerät gespeichert.
-In Mozilla Firefox werden die Daten Ende-zu-Ende-verschlüsselt bevor sie hoch geladen und auf den Servern von Mozilla gespeichert werden.
+Dieses ADD-ON nutzt den „Sync storage” des Browsers, um die Einstellungen zu speichern. Wenn der NUTZER „Sync” im Browser aktiviert, werden die Einstellungen des Add-ons hochgeladen und zwischen den Geräten, die mit dem (Mozilla) Account des Browsers verbunden sind, synchronisiert. Wenn dies nicht gemacht wird, werden die Daten lokal auf dem Gerät gespeichert.
+In Mozilla Firefox werden die Daten Ende-zu-Ende verschlüsselt, bevor sie hochgeladen und auf den Servern von Mozilla gespeichert werden.
```

Comment:
You've removed the `-`, but I would think you need to ["durchkoppel"](https://de.wikipedia.org/wiki/Durchkopplung) it… 

The noun is also written like this: https://de.wikipedia.org/wiki/Ende-zu-Ende-Verschl%C3%BCsselung

So do you have any source or so for that change?
Otherwise I'd suggest to keep it:

```suggestion
In Mozilla Firefox werden die Daten Ende-zu-Ende-verschlüsselt, bevor sie hochgeladen und auf den Servern von Mozilla gespeichert werden.
```